frontend/src/views/instructor/store/reducers.ts

```diff
@@ -0,0 +1,27 @@
+import {
+    SET_INSTRUCTOR_ACTIVE_POSITION,
+    SET_INSTRUCTOR_SELECTED_ROWS,
+} from "./constants";
+import { createReducer } from "redux-create-reducer";
+
+interface InstructorUIState {
+    selectedDdahIds: number[];
+    activePositionId: number | null;
+}
+
+// initialize the state of offer table
+const initialState: InstructorUIState = {
```

Comment:
I just love typed Redux :heart:

Reply:
It's pretty cool when it actually works!